### PR TITLE
Rewrite and reorganization of readme

### PR DESCRIPTION
**Pull request by ptrlrd**
Did some clean up and reorganization of the readme. I don't do rust a lot but I made some insinuations on how to use example directories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,20 +65,48 @@ Options:
...
+- Addressing compatibility issues with Bevy ecosystem plugins
 
-The issues I see being major in the long run are:
+### Known Issues
 
-- Arbitrary bevy ecosystem plugins not functioning properly.
-    - Since schedules are located by strings currently in canopy (`canopy_add_systems("Update", ...)`) the bevy ecosystem is incompatible with canopy mods.
-    - A possible solution could be to have a `canopy_add_plugins` method which adds all the plugins to a separate app then merges it with the main app.
+- Arbitrary Bevy ecosystem plugins may not function properly due to schedule location methods in Canopy.
+- Potential solution: Implement a `canopy_add_plugins` method to add plugins to a separate app and merge with the main app.
```
Comment (HazelTheWitch) [inline README.md]: This should be a sub-item to the last item in contributions. Currently I feel the "known issues" list is very large and a disclaimer would serve better.

**Review by HazelTheWitch — COMMENTED** [review]
Looks good, thank you for cleaning it up. I'd change the one thing mentioned and maybe add a disclaimer that it's windows only (which I forgot oops) but otherwise looks good.

File: README.md (hunk 1 above)
Comment (ptrlrd) [reply README.md]: I decided to go with "Known Issues and Disclaimers" making a broad stroke that if people start using it more they may make issues in the repo and they can be listed maybe? I'll defer to you though if you just want it to be "Disclaimers"

**Comment by HazelTheWitch** [comment]
Looks good thanks!

**Pull request merged**